UI/signin.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+    	<meta name="viewport" content="width=device-width,initial-scale=1.0">
+		<title>politico : Sign-in</title>
+		<link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" rel="stylesheet">
+		<link href="assets/css/styles.css" type="text/css" rel="stylesheet" media="all" />
+		<link href="assets/css/signin.css" type="text/css" rel="stylesheet" media="all" />
+		<link rel="shortcut icon" type="image/x-icon" href="./assets/images/logo.gif">
+	</head>
+	<body>
+
+			<nav class="politico-nav">
+					<div class="nav-container">
+						<div class="nav-left">
+							
+							<a href="./frontPage.html"><h2><img class="laravel" src="./assets/images/logo.gif" height="40px" width="40px" alt="">  POLITICO</h2></a>
+						</div>
+		
+						<div class="nav-right">
+											
+						<ul class="header">
+								<li class="login-btn">
+									<a href="./Application.html">Apply</a> 
+								</li>
+								<li class="signup-btn"> 
+								   <a href="./signup.html">sign up</a> 
+								</li>
+								<li>
+										<a href="./signin.html">
+											<button class="nav-button">Logout</button>
+										</a>
+								</li>
+							</ul>
+		
+							
+						</div>
+					</div>
+		
+					
+		
+				</nav>
+		<div class="signin-area">
+
+			<div class="signin-box">
+				<h4 class="signin-box-title">Sign-in</h4>
+				<form action="./index.html">
+					<div class="signin-input-section">
+						<input type="email" placeholder="Email" required/>
+					</div>
+					<div class="signin-input-section">
+						<input type="password" placeholder="Password" required/>
+					</div>
+					<div class="signin-submit">
+						<button type="submit">Submit</button>
+                    </div>
+              
+				</form>
+				<div class="signin-box-footer">
+					<small>
+						<a href="./signup.html">New here? &amp; Sign up</a>
+					</small>
+                </div>
+                
+                <BR></BR>
+			</div>
+        </div>
+		<footer class="politico-footer">
+				<p>politico. &copy; 2019</p>
+			</footer>
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks